dojo/product/views.py

```diff
@@ -574,7 +574,7 @@ def view_product_metrics(request, pid):
 
     product_tab = Product_Tab(prod, title=_("Product"), tab="metrics")
 
-    open_objs_by_age = {x: len([_ for _ in filters.get('open') if _.age == x]) for x in set([_.age for _ in filters.get('open')])}
+    open_objs_by_age = {x: len([_ for _ in filters.get('open') if _.age == x]) for x in {_.age for _ in filters.get('open')}}
```

Comment:
This double comprehension is an example of when removing `set()` just makes the code more difficult to read, IMO.

Reply:
Agreed from my perspective